test/framework/setup.go

```diff
@@ -275,7 +275,8 @@ func EchoServerUniversal(name, mesh, echo, token string, fs ...DeployOptionsFunc
 func IngressUniversal(mesh, token string) InstallFunc {
 	return func(cluster Cluster) error {
 		uniCluster := cluster.(*UniversalCluster)
-		app, err := NewUniversalApp(cluster.GetTesting(), uniCluster.name, AppIngress, AppIngress, true, []string{})
+		isipv6 := IsIPv6() // should we check opts as everyone else does?
```

Comment:
what's the story here?

Should we remove WithIPv6 then if we cannot override the setting from env var?
(I see this this was the only usage)

Reply:
The INgress Setup does not have opts today. WithIPv6 will be removed indeed.